examples/experimental-amp/components/Layout.js

```diff
@@ -0,0 +1,13 @@
+export default function Layout ({ children }) {
+  return (
+    <>
+      {children}
+      <style jsx global>{`
+        body {font-family: Roboto, sans-serif; padding: 30px; color: #444;}
+        h1 {margin-bottom: 5px;}
```

Comment:
Let's make these styles local, only body is really global.

Reply:
Updated